interface/src/app/gene/network/network.tpl.html

```diff
@@ -0,0 +1,15 @@
+<h1>ADAGE: Gene-Gene Network</h1>
+
+<div ng-hide="ctrl.statusMessage">
```

Comment:
If **statusMessage** is a blank string, this **div** will show up. As you may have noticed, when the gene netowork data has been retrieved from the backend successfully,  **statusMessage** will be set to a blank string.